test/Constraints/rdar68155466.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -verify %s
+
```

Comment:
I think this also needs `// REQUIRES: objc_interop`

Reply:
Right!

Reply:
✅